docker-compose.yaml

```diff
@@ -2,35 +2,50 @@ name: lumigator
 
 services:
 
-  localstack:
-    # if you want to enable local S3-style data persistence, use the following
-    # image and set LOCALSTACK_AUTH_TOKEN in your env
-    # image: localstack/localstack-pro:3.4.0
-    image: localstack/localstack:3.4.0
-    platform: linux/amd64
+  minio:
+    image: quay.io/minio/minio # TODO add version
+    command: server /data --console-address ":9001"
     ports:
-      - 4566:4566
+      - 9000:9000
+      - 9001:9001
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+      interval: 30s
+      timeout: 20s
+      retries: 3
     environment:
-      - SERVICES=s3:4566
-      - CREATE_BUCKETS=lumigator-storage
-      - LOCALSTACK_AUTH_TOKEN=${LOCALSTACK_AUTH_TOKEN}
-      - PERSISTENCE=1
-      - SNAPSHOT_SAVE_STRATEGY=ON_REQUEST
-      - EXTRA_CORS_ALLOWED_ORIGINS=*
+      - MINIO_ROOT_USER=minioadmin
+      - MINIO_ROOT_PASSWORD=minioadmin
+      # FIXME Needs to be checked
+      - MINIO_API_CORS_ALLOW_ORIGIN=*
     volumes:
-      - localstack-data:/var/lib/localstack
+    # - ${HOME}/minio/data:/data
+      - minio-data:/data
     profiles:
       - local
       - local-fe
 
-  localstack-create-bucket:
-    image: localstack/localstack:3.4.0
+  minio-admin:
+    image: ubuntu
     platform: linux/amd64
     depends_on:
-      localstack:
+      minio:
         condition: service_healthy
-    entrypoint: >
-       bash -c "awslocal s3 mb s3://lumigator-storage"
+    command:
+      - /bin/bash
+      - -c
+      - |
+          set -ex
+          apt update
+          apt install -y curl
+          curl --create-dirs -o /etc/lumigator/minio/mc https://dl.min.io/client/mc/release/linux-amd64/mc
```

Comment:
Using a generic container where we need to install the minio client requires us to install `curl` and run `apt update` first, the result is that it takes more than one minute to pull the whole system up. Another linked issue is that whenever there's an issue with `apt-get` we cannot start lumigator. This happened while testing:

<img width="583" alt="image" src="https://github.com/user-attachments/assets/d64a0ba4-0798-4faa-9bb5-7f8351566f50" />

As an alternative we could have something like:
```
 minio-admin:
    image: quay.io/minio/minio
    depends_on:
      minio:
        condition: service_healthy
    entrypoint:
      - /bin/bash
    command:
      - -c
      - |
          set -ex
          ...
```

The minio image is already cached anyway, the load time decreases to ~35 sec, and we do not have to reinstall stuff at every run. WDYT?

A follow up question: is there any way to persist the minio configuration so we can pick it up if available and run the following only if that is not already set?